app/assets/javascripts/govuk-component/tasklist.js

```diff
@@ -177,45 +177,45 @@
         event.preventDefault();
       }
 
-      function bindToggleOpenCloseAllButton(tasklistTracker) {
-        $openOrCloseAllButton = $element.find('.js-step-controls-button');
-        $openOrCloseAllButton.on('click', function () {
-          var shouldOpenAll;
+      function bindToggleShowHideAllButton(tasklistTracker) {
+        $showOrHideAllButton = $element.find('.js-step-controls-button');
+        $showOrHideAllButton.on('click', function () {
+          var shouldshowAll;
 
-          if ($openOrCloseAllButton.text() == bulkActions.openAll.buttonText) {
-            $openOrCloseAllButton.text(bulkActions.closeAll.buttonText);
-            $element.find('.js-toggle-link').text(bulkActions.closeAll.linkText)
-            shouldOpenAll = true;
+          if ($showOrHideAllButton.text() == bulkActions.showAll.buttonText) {
+            $showOrHideAllButton.text(bulkActions.hideAll.buttonText);
+            $element.find('.js-toggle-link').text(bulkActions.hideAll.linkText)
+            shouldshowAll = true;
 
-            tasklistTracker.track('pageElementInteraction', 'tasklistAllOpened', {
-              label: bulkActions.openAll.eventLabel + ": " + taskListSize
+            tasklistTracker.track('pageElementInteraction', 'tasklistAllShown', {
+              label: bulkActions.showAll.eventLabel + ": " + taskListSize
             });
           } else {
-            $openOrCloseAllButton.text(bulkActions.openAll.buttonText);
-            $element.find('.js-toggle-link').text(bulkActions.openAll.linkText)
-            shouldOpenAll = false;
+            $showOrHideAllButton.text(bulkActions.showAll.buttonText);
+            $element.find('.js-toggle-link').text(bulkActions.showAll.linkText)
+            shouldshowAll = false;
 
-            tasklistTracker.track('pageElementInteraction', 'tasklistAllClosed', {
-              label: bulkActions.closeAll.eventLabel + ": " + taskListSize
+            tasklistTracker.track('pageElementInteraction', 'tasklistAllHidden', {
+              label: bulkActions.hideAll.eventLabel + ": " + taskListSize
             });
           }
 
-          setAllStepsOpenState(shouldOpenAll);
-          $openOrCloseAllButton.attr('aria-expanded', shouldOpenAll);
-          setOpenCloseAllText();
+          setAllStepsShownState(shouldshowAll);
+          $showOrHideAllButton.attr('aria-expanded', shouldshowAll);
+          setShowHideAllText();
           setHash(null);
 
           return false;
         });
       }
 
-      function setOpenCloseAllText() {
-        var openSteps = $element.find('.step-is-open').length;
+      function setShowHideAllText() {
+        var shownSteps = $element.find('.step-is-shown').length;
```

Comment:
See above, state is in DOM, is there a way to have this on the instance of this component?